packages/itensor/quri_parts/itensor/estimator.py

```diff
@@ -143,9 +143,10 @@ def create_itensor_mps_concurrent_estimator(
     """Returns a :class:`~ConcurrentQuantumEstimator` that uses ITensor MPS
     simulator to calculate expectation values.
 
-    For now, this function works when the executor is defined like below
-    `with ProcessPoolExecutor(max_workers=2,
-    mp_context=get_context("spawn")) as executor:`
+    For now, this function works when the executor is defined like below::
+
+        with ProcessPoolExecutor(max_workers=2, mp_context=get_context("spawn"))
+        as executor:
```

Comment:
It should be like this:

```suggestion
        with ProcessPoolExecutor(
            max_workers=2, mp_context=get_context("spawn")
        ) as executor:
```